core/shared/src/main/scala/zio/Schedule.scala

```diff
@@ -82,7 +71,7 @@ object Schedule {
   /**
    * See [[ZSchedule.delayed]]
    */
-  final def delayed[A](s: Schedule[A, Duration]): Schedule[A, Duration] =
+  final def delayed[A](s: Schedule[A, Duration]): ZSchedule[zio.clock.Clock, A, Duration] =
```

Comment:
No schedule can go into `Schedule` companion object unless `R = Any`. Otherwise you could use `Schedule.XYZ` to create a `ZSchedule`.

If it turns out most of these / many of these can no longer be placed here, we might just rename `ZSchedule` => `Schedule` and put everything in one location...

Reply:
Oh, I missed that one.
Yeah I believe removing the Schedule/ZSchedule split makes sense here. I'll do it in a seperate follow up pr